kiota_abstractions/default_query_parameters.py

```diff
@@ -12,7 +13,26 @@ class QueryParameters:
     """
     Default placeholder class for query parameters.
     """
-    pass
+    # Include count of items
+    count: Optional[bool] = None
+
+    # Expand related entities
+    expand: Optional[List[str]] = None
+
+    # Filter items by property values
+    filter: Optional[str] = None
+
+    # Order items by property values
+    orderby: Optional[List[str]] = None
+
+    # Select properties to be returned
+    select: Optional[List[str]] = None
+
+    # Skip the first n items
+    skip: Optional[int] = None
+
+    # Show only the first n items
+    top: Optional[int] = None
```

Comment:
I think we should remove those as :
- this makes an assumption of an OData API
- this makes the assumption those query parameters are supported which could lead the application developer to think so when it's not the case

Reply:
I concur on the two points.

> My concern here is that since we're not using generic parameters when they type the Python equivalent of
client.users.get(o => o.QueryParameters.Select), they won't get auto-completion of Select here.

But now I'm not sure what you mean by `generic parameters`.

Reply:
By that I mean a type parameter for the request option class, or a template if we talk C++.